tools/astarte_device_fleet_simulator/lib/astarte_device_fleet_simulator/config.ex

```diff
@@ -0,0 +1,178 @@
+  app_env(:ignore_ssl_errors, :astarte_device_fleet_simulator, :ignore_ssl_errors,
+    os_env: "DEVICE_FLEET_IGNORE_SSL_ERRORS",
+    type: :boolean,
+    default: false
+  )
+
+  @envdoc "Astarte Broker URL (e.g. https://broker.astarte.example.com/)."
+  app_env(:broker_url, :astarte_device_fleet_simulator, :broker_url,
+    os_env: "DEVICE_FLEET_BROKER_URL",
+    type: :binary,
+    required: true
+  )
+
+  @envdoc "Realm name."
+  app_env(:realm, :astarte_device_fleet_simulator, :realm,
+    os_env: "DEVICE_FLEET_REALM",
+    type: :binary,
+    required: true
+  )
+
+  @envdoc "The Astarte JWT employed to access Astarte APIs. The token can be generated with: `$ astartectl utils gen-jwt <service> -k <your-private-key>.pem`."
+  app_env(:jwt, :astarte_device_fleet_simulator, :jwt,
+    os_env: "DEVICE_FLEET_JWT",
+    type: :binary,
+    required: true
+  )
+
+  @envdoc "Time interval between consecutive spawns of Astarte devices (in seconds)."
+  app_env(:spawn_interval_ms, :astarte_device_fleet_simulator, :spawn_interval_ms,
+    os_env: "DEVICE_FLEET_SPAWN_INTERVAL_MILLISECONDS",
+    type: :pos_integer,
+    default: 1000
+  )
+
+  @envdoc "Time interval between messages from a single Astarte devices (in milliseconds)."
+  app_env(:publication_interval_ms, :astarte_device_fleet_simulator, :publication_interval_ms,
+    os_env: "DEVICE_FLEET_PUBLICATION_INTERVAL_MILLISECONDS",
+    type: :integer,
+    default: 1000
+  )
+
+  @envdoc "The number of Astarte device forming a test fleet."
+  app_env(:device_count, :astarte_device_fleet_simulator, :device_count,
+    os_env: "DEVICE_FLEET_DEVICE_COUNT",
+    type: :pos_integer,
+    default: 10
+  )
+
+  @envdoc "The length of the test (in seconds)."
+  app_env(:test_duration_s, :astarte_device_fleet_simulator, :test_duration_s,
+    os_env: "DEVICE_FLEET_TEST_DURATION_SECONDS",
+    type: :pos_integer,
+    default: 30
+  )
+
+  @envdoc "The path of the interface to which data are sent."
+  app_env(:path, :astarte_device_fleet_simulator, :path,
+    os_env: "DEVICE_FLEET_PATH",
+    type: :binary,
+    default: "/streamTest/value"
+  )
+
+  @envdoc "The value to send."
+  app_env(:value, :astarte_device_fleet_simulator, :value,
+    os_env: "DEVICE_FLEET_VALUE",
+    type: :float,
+    default: 0.3
+  )
+
+  @envdoc "The QoS mode for messages sent from Astarte devices."
+  app_env(:qos, :astarte_device_fleet_simulator, :qos,
+    os_env: "DEVICE_FLEET_QOS",
+    type: :pos_integer,
+    default: 2
+  )
+
+  @envdoc "Allow already connected Astarte devices to send messages while others are still connecting. Defaults to false."
+  app_env(
+    :allow_messages_while_spawning,
+    :astarte_device_fleet_simulator,
+    :allow_messages_while_spawning,
+    os_env: "DEVICE_FLEET_ALLOW_MESSAGES_WHILE_SPAWINING",
+    type: :boolean,
+    default: false
+  )
+
+  @envdoc "Fleet simulator instance ID, used when deploying multiple fleet simulator instances. Defaults to `astarte-fleet-simulator`"
```

Comment:
It is always used to generate a valid device-id. Rephrase to highlight that it is useful to avoid id collisions when deploying multiple instances.